docs/tessellation/usage/analysis/rotating_bar.md

```diff
@@ -57,8 +57,16 @@ dt = 0.01 * u.Gyr
 steps = 500
 omega = 30 * u.km / u.s / u.kpc
 
-tanal = TessellationAnalysis2D(initial_condition, values, potential_definition,
-                             dt, steps, pattern_speed=omega, chunksize=500, mpchunksize=20)
+tanal = TessellationAnalysis2D(
+    initial_condition,
+    values,
+    potential_definition,
+    dt,
+    steps,
+    pattern_speed=omega,
+    pidgey_chunksize=500,
+    mp_chunksize=20,
+)
 tanal.save("no_bar_example.hdf5")
```

Comment:
I did have some trouble with this line in a couple tests (in retrieving the source code to save into the output) -- but thinking it was specific to my build. Anyone else find an issue? If so, I'll report more formally.

Reply:
What tests were causing issues?